pkgs/servers/alice-lg/default.nix

```diff
@@ -0,0 +1,85 @@
+{ lib
+, fetchFromGitHub
+, buildGoModule
+, fetchYarnDeps
+, stdenv
+, yarn
+, nodejs
+, git
+, fixup_yarn_lock
+}:
+
+buildGoModule rec {
+  pname = "alice-lg";
+  version = "6.0.0";
+
+  src = fetchFromGitHub {
+    owner = "alice-lg";
+    repo = "alice-lg";
+    rev = version;
+    hash = "sha256-BdhbHAFqyQc8UbVm6eakbVmLS5QgXhr06oxoc6vYtsM=";
+  };
+
+  vendorSha256 = "sha256-SNF46uUTRCaa9qeGCfkHBjyo4BWOlpRaTDq+Uha08y8=";
+
+  ui = stdenv.mkDerivation {
```

Comment:
This should be added to the passthru `passthru.ui = ui;`

Reply:
When trying to do that nix-build complains about it not finding the ui attribute:
```
nix-build -A alice-lg
error: undefined variable 'ui'

       at /home/janik/Projects/nixos/nixpkgs/pkgs/servers/alice-lg/default.nix:71:13:

           70|   preBuild = ''
           71|     cp -R ${ui}/ ui/build/
             |             ^
           72|   '';
```

but I'm quite unfamiliar with passthru and the [manual](https://nixos.org/manual/nixpkgs/stable/#special-variables) isn't to helpful with this.

Reply:
You need to reference `passthru.ui` as you're in a `rec` block, you can reference all the others attributes.
Otherwise, using functional mkDerivation, you can have `finalAttrs.ui` (grep the repo for examples)

Reply:
Okay done